Spring loaded anti backlash nut update

StephenGitHuber opened the pull request:
The one note I haven't addressed is:

Steps 5,6,7,8 put in backwards, video too. Explain to flip it, we found a better way

Chatted with Jay and he doesn't think it matters, just the Z axis has to be facing down. All of our assembly docs show this way. Is it worth changing all the pics?

SienciLabs commented:
@StephenGitHuber I'm not sure I understand your original comment, could you re-explain?

Regarding the writing of this page I think the work you did on adding all the arrows really adds lots of clarity and the picture quality is great. Couple of notes after review:

1. The new lines on the steps in the Y-axis section was causing the numbering to not work so I fixed it
2. The X-axis and Z-axis sections didn't have numbered steps. I added for X can you also add for Z (make sure spacing is correct to get numbering working too)?
3. Could you make another pass at the flow of the instructions just to check that all the steps are clear. I was going through the Y-axis section and even with the pictures was still getting a bit confused on what to do - I made some updates but stopped about halfway through Y to hand back if you want to give it another look.
4. It might be a good idea at the bottom of the page to add a troubleshooting section since I recall there was instances of people having issues with the nuts and Jay recommending a wear-in cycle. It might also be nice just to add a conclusion at the end of assembly to say "hey you're done now!"

As always you can always see my tweaks in the timeline above. These were my changes today: https://github.com/Sienci-Labs/Resources/pull/28/commits/2d8657289a9318e4b18d163626c46b8afe555a3b

(outside of my comments, I just added some extra flair to the start in case people wanted to read more about the work done on the design of the nuts)

StephenGitHuber commented:
Ok, I've reviewed all the points you raised, updated the steps for a bit more clarity, tweaked a couple pics, renamed all pics and file, added a troubleshooting section after chatting with Jay a bit more and some community links.

I've kept this a draft and skip file for now. Should be getting really close to posting this one @SienciLabs

SienciLabs reviewed (approved):
Changes were in a great direction, made some other edits and should be ready to post I think
